interbtc-spec/docs/source/spec/vault-registry.rst

```diff
@@ -638,11 +638,7 @@ Specification
 
 *Returns*
 
-* A tupple of the new total ``toBeReplacedTokens`` and ``replaceCollateral``.
-  
-*Events*
```

Comment:
Why did you remove this event?

Reply:
Wasn't specified below, added in a separate commit. Also included a missing event here and in the implementation, see: https://github.com/interlay/interbtc/pull/191